src/components/EmojiPicker/EmojiPickerMenu/useEmojiPickerMenu.ts

```diff
@@ -23,12 +24,14 @@ const useEmojiPickerMenu = () => {
     const [preferredSkinTone] = usePreferredEmojiSkinTone();
     const {windowHeight} = useWindowDimensions();
     const StyleUtils = useStyleUtils();
+    const {keyboardHeight} = useKeyboardState();
+
     /**
      * At EmojiPicker has set innerContainerStyle with maxHeight: '95%' by styles.popoverInnerContainer
      * to avoid the list style to be cut off due to the list height being larger than the container height
      * so we need to calculate listStyle based on the height of the window and innerContainerStyle at the EmojiPicker
      */
-    const listStyle = StyleUtils.getEmojiPickerListHeight(isListFiltered, windowHeight * 0.95);
+    const listStyle = StyleUtils.getEmojiPickerListHeight(isListFiltered, windowHeight * 0.95 - keyboardHeight);
```

Comment:
@huult, mind updating the comment above to explain the keyboardHeight that we added here?

Reply:
it's done